### PR TITLE
[*] example corrections

### PR DESCRIPTION
**Pull request by dusanDrabik**

### CODE REVIEW

File: src/examples/ContentDetail/contentDetail.js [hunk 1]
```diff
@@ -15,27 +15,27 @@ const contentDetail = document.getElementById('contentDetail');
 
 let contentItems = [];
```
Comment (peter-rybar) [inline src/examples/ContentDetail/contentDetail.js]: create const

File: src/examples/ContentDetail/contentDetail.js [hunk 2]
```diff
@@ -15,27 +15,27 @@ const contentDetail = document.getElementById('contentDetail');
...
-        .catch(err => {
-            console.error(err);
-            contentList.innerHTML = 'Error loading content';
-        });
-}
+contentList.innerHTML = 'Loading ...';
+decent.content().searchContent(new decent.SearchParams())
+    .then(content => {
+        contentList.innerHTML = renderContent(content);
+        contentItems = content;
```
Comment (peter-rybar) [inline src/examples/ContentDetail/contentDetail.js]: what about do it more typescript way
contentItems.push(...content);

File: src/examples/ContentDetail/contentDetail.js [hunk 3]
```diff
@@ -15,27 +15,27 @@ const contentDetail = document.getElementById('contentDetail');
...
 
 function renderContent(content) {
-    let render = '<ul>';
-     render += content.map(c => '<li onclick="showDetail(\'' + c.id + '\')"><a href="#">' + c.synopsis.title + '</a></li>');
-     render += '</ul>';
-     if (content.length === 0) {
-         render = '<h3>No content</h3>';
-     }
-     return render
+    let render = '';
```
Comment (peter-rybar) [inline src/examples/ContentDetail/contentDetail.js]: what about improve readability and performance:

const render = [];
render.push('<h3>No content</h3>');
...
return render.join("");

**Review by peter-rybar — CHANGES_REQUESTED** [review]

File: src/examples/ContentDetail/contentDetail.js (hunk 1 above)
Comment (dusanDrabik) [reply src/examples/ContentDetail/contentDetail.js]: Nope, there is changed array in there later in call, not appended.

File: src/examples/ContentDetail/contentDetail.js (hunk 3 above)
Comment (dusanDrabik) [reply src/examples/ContentDetail/contentDetail.js]: On the way

**Comment by dusanDrabik** [comment]
https://decentplatform.atlassian.net/browse/DGO2-170

**Review by peter-rybar — COMMENTED** [review]
FAIL  test/crypt.test.ts
  ● Test suite failed to run

    Cannot find module 'decentjs-lib' from 'crypt.test.ts'
      
      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:191:17)
      at Object.<anonymous> (test/crypt.test.ts:3:20)

 FAIL  test/utils.test.ts
  ● Test suite failed to run

    Cannot find module 'decentjs-lib' from 'utils.test.ts'
      
      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:191:17)
      at Object.<anonymous> (test/utils.test.ts:2:20)

 FAIL  test/chainApi.test.ts

**Comment by dusanDrabik** [comment]
Dev dependency for decentjs-lib is already merged in develop. This two PRs was posted at same time, therefore decentjs-lib dev dependency is not presented in this branch.

**Pull request merged**